APISubmissionGuide.md

```diff
@@ -11,46 +11,13 @@
 ## DABS Submission Process
 
 ### Upload A, B, C Files
-- Step 1: call `/v1/submit_files/` (POST) to create the submission
-    - Header:
-        - `X-Session-ID`: string, session token id
-    - Payload:
-        - `appropriations`: string, file A name
-        - `program_activity`: string, file B name
-        - `award_financial`: string, file C name
-        - `cgac_code`: string, CGAC of agency (null if FREC agency)
-        - `frec_code`: string, FREC of agency (null if CGAC agency)
-        - `is_quarter`: boolean (true for quarterly submissions)
-        - `reporting_period_start_date`: string, starting date of submission (MM/YYYY)
-        - `reporting_period_end_date`: string, ending date of submission (MM/YYYY)
-    - **NOTE**: for monthly submissions, start/end date are the same
-    - Response:
-        - `appropriations_id`: int, ID of file A upload
-        - `appropriations_key`: string, path to file A within S3 bucket
-        - `program_activity_id`: int, ID of file B upload
-        - `program_activity_key`: string, path to file B within S3 bucket
-        - `award_financial_id`: int, ID of file C upload
-        - `award_financial_key`: string, path to file C within S3 bucket
-        - `award_procurement_id`: int, ID of file D1 upload
-        - `award_procurement_key`: string, path to file D1 within S3 bucket
-        - `award_id`: int, ID of file D2 upload
-        - `award_key`: string, path to file D2 within S3 bucket
-        - `executive_compensation_id`: int, ID of file E upload
-        - `executive_compensation_key`: string, path to file E within S3 bucket
-        - `sub_award_id`: int, ID of file F upload
-        - `sub_award_key`: string, path to file F within S3 bucket
-        - `bucket_name`: string, name of bucket on S3 that files are stored in
-        - `credentials`: object, the credentials to S3 (AccessKeyId, Expiration, SecretAccessKey, SessionToken)
-        - `submission_id`: int, ID of the submission that was created
-- Step 2: Upload A, B, C files to S3 (Process to be defined)
-- Step 3: Call `/v1/finalize_job/` (POST) for each file (A, B, C)
-    - Payload:
-        - `upload_id`: int, ID of the file upload received from the `submit_files` response
-    - Response:
-        - `success`: boolean, successful or failed progression of the jobs
+- Step 1: call `/v1/submit_files/` (POST) to create the submission.
+- For details on its use, click [here](./dataactbroker/README.md#post-v1submit_files)
 
 ### Validate A, B, C Files
-- File-level validation begins automatically on completion of `finalize_job` call
+- File-level validation begins automatically:
```

Comment:
I'm sorry, I was mistaken about this before. In the API submission guide we just want what happens from the API submitter's perspective, not the frontend part.

Reply:
Fixed